src/demo/DemoTestLoanService.java

```diff
@@ -2348,4 +2351,122 @@ private static LoanAccount updateExpectedDisbursementDateAndFirstRepaymentDate(L
 		return updatedLoanAccount;
 	}
 
+	/**
+	 * Tests updating a loan in order to set a settlement account on it. It works only with Loans having Account Linking
+	 * enabled
+	 * 
+	 * @throws MambuApiException
+	 */
+
+	public static void testPatchSetllementAccounts() throws MambuApiException {
```

Comment:
Since this method performs two operations on loan accounts - setting a settlement and also deleting it afterwards, I believe it would be better to change its name to something more meaningful, a name that should reflect what actually happens inside of the method.